stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/messages/composer/MessageComposer.kt

```diff
@@ -567,6 +567,7 @@ internal fun DefaultComposerLabel(ownCapabilities: Set<String>) {
     Text(
         text = text,
         color = ChatTheme.colors.textLowEmphasis,
+        style = ChatTheme.messageComposerTheme.inputField.textStyle
```

Comment:
Looks good!
Can you fix spotless and put a comma at the end of this line?
Thanks,